packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -0,0 +1,240 @@
+import type { FramingBehavior } from "core/Behaviors/Cameras/framingBehavior";
+import { ArcRotateCamera } from "core/Cameras/arcRotateCamera";
+import type { AbstractEngine } from "core/Engines";
+import { HemisphericLight } from "core/Lights/hemisphericLight";
+import { SceneLoader } from "core/Loading/sceneLoader";
+import { CubeTexture } from "core/Materials/Textures/cubeTexture";
+import { Texture } from "core/Materials/Textures/texture";
+import { Color4 } from "core/Maths/math";
+import { Vector3 } from "core/Maths/math.vector";
+import type { AssetContainer } from "core/assetContainer";
+import type { IDisposable } from "core/scene";
+import { Scene } from "core/scene";
+import type { Nullable } from "core/types";
+
+// TODO: Dynamic imports?

```

Comment:
yes!
Small note - typescript will read the types synchronously, so ts will claim the functions are already available though it will only be available after the module is actually imported.

Reply:
Cool. My preference is to introduce the dynamic imports in a separate PR. If you feel strongly it needs to be done in the initial commit, please ping me. :)